deploy.py

```diff
@@ -36,129 +37,137 @@ def deploy(bento_bundle_path, deployment_name, config_json):
+                "create",
+                "--name",
+                storage_account_name,
+                "--resource-group",
+                resource_group_name,
+            ]
+        )
+    console.print(f"Created Azure storage account {storage_account_name}")
 
-    print(f"Creating Azure function plan {function_plan_name}")
-    run_shell_command(
-        [
-            "az",
-            "functionapp",
-            "plan",
-            "create",
-            "--name",
-            function_plan_name,
-            "--resource-group",
-            resource_group_name,
-            "--is-linux",
-            "--sku",
-            azure_config["function_sku"],
-            "--min-instances",
-            str(azure_config["min_instances"]),
-            # Only for EP plans
-            # "--max-burst",
-            # str(azure_config["max_burst"]),
-        ]
-    )
+    with console.status(f"Creating Azure function plan"):
+        run_shell_command(
+            [
+                "az",
+                "functionapp",
+                "plan",
+                "create",
+                "--name",
+                function_plan_name,
+                "--resource-group",
+                resource_group_name,
+                "--is-linux",
+                "--sku",
+                azure_config["function_sku"],
+                "--min-instances",
+                str(azure_config["min_instances"]),
+                # Only for EP plans
+                # "--max-burst",
+                # str(azure_config["max_burst"]),
+            ]
+        )
+    console.print(f"Created Azure function plan {function_plan_name}")
 
-    print(f"Creating Azure ACR {acr_name}")
-    run_shell_command(
-        [
-            "az",
-            "acr",
-            "create",
-            "--name",
-            acr_name,
-            "--sku",
-            azure_config["acr_sku"],
-            "--resource-group",
-            resource_group_name,
-        ]
-    )
+    with console.status(f"Creating Azure ACR"):
+        run_shell_command(
+            [
+                "az",
+                "acr",
+                "create",
+                "--name",
+                acr_name,
+                "--sku",
+                azure_config["acr_sku"],
+                "--resource-group",
+                resource_group_name,
+            ]
+        )
 
-    # build and push docker
-    run_shell_command(
-        [
-            "az",
-            "acr",
-            "login",
-            "--name",
-            acr_name,
-            "--resource-group",
-            resource_group_name,
-        ]
-    )
+        # build and push docker
+        run_shell_command(
+            [
+                "az",
+                "acr",
+                "login",
+                "--name",
+                acr_name,
+                "--resource-group",
+                resource_group_name,
+            ]
+        )
+    console.print(f"Created Azure ACR {acr_name}")
```

Comment:
can you join this entire block in 1 console.status says "Create resources in azure" and below that each console.print will tell the various resources created after they are created. Does that make sense?